### ISSUE TITLE
How to stop recordings ?

### ISSUE THREAD

**Issue by JinJon4000** [2MinhetNguyles/Chaturbate-Recorder-GUI#9]
Hi,
I saw in the Running Recorders window that to stop recordings, we had to wait that the model stops OR "to show the window, click on it and press Q". But I didn't manage to do it, what do you call "the window", which window ?
I pressed Q being in the Running Recorders window and also in the main window but nothing happened.
I had to go to the task manager and end the ffmpeg process to stop the recording and the mp4 file was not readable. I guess that's not the good way to stop recordings.
You could maybe add a STOP button (for each recording if there are several).
Thanks for the support.

**Comment by Alienmoon** [comment]
I have to agree with a stop button, but what confuses me is when you show all recordings, they are marked by a few numbers,... It'll be nice to understand which recording is for what model.

Once the Black command window opens, you click it and then once selected, press "Q" to stop that recording... though its been over a year, I'm sure you've sussed this by now, lol
